src/views/illness/illness-start-date.njk

```diff
@@ -0,0 +1,111 @@
+{% extends 'layout.njk' %}
+
+{% from 'govuk/components/back-link/macro.njk' import govukBackLink %}
+{% from 'govuk/components/error-summary/macro.njk' import govukErrorSummary %}
+{% from 'govuk/components/button/macro.njk' import govukButton %}
+{% from 'govuk/components/date-input/macro.njk' import govukDateInput %}
+
+{% block pageTitle %}
+   Appeal due to ill health: Tell us when the illness started
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <form method="post">
+
+        {{
+          govukErrorSummary ({
+            titleText: 'There is a problem',
+            errorList: validationResult.errors
+          }) if validationResult and validationResult.errors.length > 0
+        }}
+
+        {% set classes_day = 'govuk-input--width-2' %}
+        {% set classes_month = 'govuk-input--width-2' %}
+        {% set classes_year = 'govuk-input--width-4' %}
+
+        {% if isStartDateDayErrorFlag %}
```

Comment:
I feel this and controller logic are super complex. Would't that be easier to just do `validationResult.getErrorForField('startDay') or validationResult.getErrorForField('startDate')` in template?

Reply:
i've removed the flags and refactored logic in controller/template